lib/vizkit/widgets/task_inspector/task_inspector.rb

```diff
@@ -18,15 +18,18 @@ def self.create_widget(parent = nil)
 
         #populate widget menu
         Vizkit.default_loader.plugin_specs.keys.sort.each do |name|
-            # do not add qt base widgets
-            next if name[0] == "Q" && Qt.const_defined?(name[1..-1])
-            action = form.menuWidgets.addAction(name)
-            action.connect SIGNAL("triggered()") do
-                w = Vizkit.default_loader.create_plugin name
-                w.show if w.respond_to?(:show) || w.kind_of?(Qt::Widget)
+            begin
+                # do not add qt base widgets
+                next if name[0] == "Q" && Qt.const_defined?(name[1..-1])
+                action = form.menuWidgets.addAction(name)
+                action.connect SIGNAL("triggered()") do
+                    w = Vizkit.default_loader.create_plugin name
+                    w.show if w.respond_to?(:show) || w.kind_of?(Qt::Widget)
+                end
+            rescue
```

Comment:
I'm using QwtThermo widget. But it's always searching for wtThermo widget and crashed when trying to add it